platforms/access-bridge/src/services/access-service.ts

```diff
@@ -0,0 +1,118 @@
+import jwt from 'jsonwebtoken';
+import {
+  AccessTokensResponse,
```

Comment:
Is there a reason to define these types external to this file? It seems they are only used in this code and could be better encapsulated by putting them here.

Reply:
Some of them are required on the FE part as well, e.g. the PassportResponse, but stripped off the non-required ones, so we are good.